dev-libs/botan/botan-2.6.0.recipe

```diff
@@ -0,0 +1,93 @@
+SUMMARY="A cryptography library written in C++"
+DESCRIPTION="This is the best option for crypthography in C++"
```

Comment:
I would prefer a more neutral description.

Reply:
I'll update with the given comments :) (ps, most of them were left overs from previous version) :)

Reply:
Cool!
Yep :)
BTW, shall we drop the previous recipe?

Reply:
Once this one is ok I'm fine with it :)